docs/tutorials/python/predict_image.md

```diff
@@ -57,8 +57,8 @@ from collections import namedtuple
 Batch = namedtuple('Batch', ['data'])
 
 def get_image(url, show=False):
-    # download and show the image
-    fname = mx.test_utils.download(url)
+    # download and show the image. Remove query string from the file name.
+    fname = mx.test_utils.download(url, fname=url.split('/')[-1].split('?')[0])
```

Comment:
should you make sure it's not empty in case url is 'https://foo.bar/?raw=1'

Reply:
even though it's really rare to happen

Reply:
Yeah, I don't think it will happen here, because the fix is done only for this particular tutorial - all urls are known in advance.